src/anemoi/training/data/dataset.py

```diff
@@ -55,9 +56,11 @@ def __init__(
             Shuffle batches, by default True
         label : str, optional
             label for the dataset, by default "generic"
-
+        effective_bs : int, default 1
```

Comment:
Why does effective bs need to be introduced for transfer learning?

Reply:
In order to effectively reproduce other experiments (eg. Met-Norway) we need to follow the same training regime. 
So training with 10 epochs with bs=32 will require to know exactly the effective batch size to make the experiment reproducible.